examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java

```diff
@@ -30,7 +30,7 @@ protected List<ReactPackage> getPackages() {
           @SuppressWarnings("UnnecessaryLocalVariable")
           List<ReactPackage> packages = new PackageList(this).getPackages();
           // Packages that cannot be autolinked yet can be added manually here, for example:
-          // packages.add(new MyReactNativePackage());
+          packages.add(new ReactVideoPackage());
```

Comment:
@KrzysztofMoch This one was missing ! but now it work fine, thank uyou for the tips of removing react-native-video from package.json !